src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -80,6 +181,29 @@ def regenerate_credential(cmd, client, name, id_, resource_group_name=None):
     return __convert_api_key_to_json(credentail, endpoint)
 
 
+def __get_resource_identity(assign_identity):
+    system_assigned = False
+    user_assigned = {}
+    for identity in assign_identity:
+        if identity == SYSTEM_ASSIGNED_IDENTITY:
+            system_assigned = True
+        else:
+            user_assigned[identity] = UserIdentity()
+
+    if system_assigned:
+        identity_type = SYSTEM_ASSIGNED
+    elif user_assigned:
+        identity_type = USER_ASSIGNED
+    else:
+        identity_type = "None"
+
+    if system_assigned and user_assigned:
+        identity_type = SYSTEM_USER_ASSIGNED
```

Comment:
nit:

    if system_assigned and user_assigned:
        identity_type = SYSTEM_USER_ASSIGNED
    elif system_assigned:
        identity_type = SYSTEM_ASSIGNED
    elif user_assigned:
        identity_type = USER_ASSIGNED
    else:
        identity_type = "None"